src/group_operations/multiplication_operation.jl

```diff
@@ -162,7 +162,7 @@ diff_left_compose(::LieGroup{𝔽, <:AbstractMultiplicationGroupOperation}, g, h
 function diff_left_compose!(
         G::LieGroup{𝔽, <:AbstractMultiplicationGroupOperation}, Y, g, h, X
     ) where {𝔽}
-    return copyto!(LieAlgebra(G), Y, g * X)
+    return copyto!(LieAlgebra(G), Y, X)
```

Comment:
https://github.com/JuliaManifolds/Manifolds.jl/blob/a4834aedcc7f7d74b981a3f8dd2e819536040300/src/groups/group.jl#L1038-L1049

Reply:
Check that carefully, as far as I see there, we are interested in `Y` so I think it is not even the other way around - at least only looking at `Y`.

Reply:
but it might be worth adding this to the interface/generics docs of `diff_left_compose` and `diff_right_compose`, maybe even in this PR since it handles those two.

Reply:
So I checked this carefully and the proposed fix is not correct here, because that would mean the composition had the identity as derivative and that does not make sense.

Reply:
Ah no, I see now.. I add the documentation on the other branch (since I just had that checked out. Your fixes are correct.

Reply:
Eh yes it is the other way around than in your code. I now checked the original derivation and added it to the docs. Please also update the docs here accordingly then and not just the code.

Reply:
Yes, it looks like it was swopped then. The `LeftForward` `RightBackward` was a bit confusing.